### PR TITLE
Improve cli colors

### PR DESCRIPTION
**Pull request by dgzlopes**
- Add ```host_ip``` pattern.
- Fix random colors. 
   * e.g:
shipping

Signed-off-by: Daniel González Lopes <danielgonzalezlopes@gmail.com>

### CODE REVIEW

File: powerfulseal/cli/pscmd.py [hunk 1]
```diff
@@ -53,7 +54,7 @@ def colour_output(output, extras=None):
     else:
         pattern = DEFAULT_COLOR_KEYWORDS
     for word, colour in pattern.items():
-        output = output.replace(word, colored(word, colour))
+        output = output.replace(word+'=', colored(word, colour)+'=')
```
Comment (seeker89) [inline powerfulseal/cli/pscmd.py:57]: Isn't this going to stop coloring UP, DOWN and Running?

Comment (seeker89) [reply powerfulseal/cli/pscmd.py:57]: I guess if going this way, you could also add a `=word` variant.

Comment (dgzlopes) [reply powerfulseal/cli/pscmd.py:57]: Even if I added the `=word` pattern, UP and DOWN  would still not work... (NodeState.UP).

Maybe we can add NodeState to the UP and DOWN keywords.

Comment (seeker89) [reply powerfulseal/cli/pscmd.py:57]: btw https://github.com/bloomberg/powerfulseal/commit/94343afe5e213f3db860491eccd33c2a70c31f28

**Pull request merged**